ceph.spec.in

```diff
@@ -61,6 +78,11 @@ Requires: systemd
 %if 0%{with cephfs_java}
 BuildRequires: sharutils
 %endif
+%if 0%{with selinux}
+BuildRequires:	checkpolicy
+BuildRequires:	selinux-policy-devel
+BuildRequires:	/usr/share/selinux/devel/policyhelp
```

Comment:
I do not need to have mls enabled for this. I was trying to run this on a range of a more recent fedoras (21+) and rhel/centos 7 only.

Anyway, did the installation of that package fixed your issue? If yes then we could probably add it to the dep list, at least for rhel6-.